osi_common.proto

```diff
@@ -338,6 +338,45 @@ message Spherical3d
     optional double elevation = 3;
 }
 
+//
+// \brief Assignment of an object to a logical lane
+//
+// An object is assigned to a logical lane if it overlaps the logical lane.
+// Assignment happens even if the reference point is outside the lane, and only
+// a part of the object overlaps.
+//
+// As an exception to this, \c TrafficSign and \c TrafficLight are assigned to
+// a logical lane if they control traffic on that lane.
+//
+message LogicalLaneAssignment
+{
+    // ID of the LogicalLane the object is assigned to.
+    //
+    optional Identifier assigned_lane_id = 1;
+
+    // S position of the object reference point on the lane, in the ST
+    // coordinate system of the lane.
+    //
+    // s_position might be outside [s_start,s_end] of the lane (and even
+    // outside [startS,endS] of the reference line) if the reference point is
+    // outside the lane, but the object overlaps the lane or a TrafficSign or
+    // TrafficLight is assigned to a lane.
+    //
+    optional double s_position = 2;
+
+    // T position of the object reference point on the lane, in the ST
+    // coordinate system of the lane.
```

Comment:
s/t of the lane or the reference line?

Reply:
The ST coordinate system is defined by the reference line, so these are identical. I can reword to refer to the reference line.